templates/catalog/suppliers.tpl

```diff
@@ -5,5 +5,5 @@
 {extends file='catalog/brands.tpl'}
 
 {block name='brand_header'}
-  <h1>{l s='Suppliers' d='Shop.Theme.Catalog'}</h1>
+  <h1 class="h1 mb-4">{l s='Suppliers' d='Shop.Theme.Catalog'}</h1>
```

Comment:
isn't it the job for `templates/components/section-title.tpl`? I may be wrong

Reply:
@JBLach @NeOMakinG I think there should not be a need of adding the `h1` class. `h1` tag and `.h1` class should be the same I think no?

Reply:
> @JBLach @NeOMakinG I think there should not be a need of adding the `.h1` class. `h1` tag and `.h1` class should be the same I think no?

You're right, we might introduced some wrong style somewhere if it's the case